### PR TITLE
Restore 'realsvc:test'

### PR DESCRIPTION
**Pull request by DLehenbauer**

### CODE REVIEW

**Review by anthony-murphy — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -62,7 +62,7 @@
     "test:mocha:report": "lerna run test:mocha --stream --no-bail --no-sort -- -- --timeout 4s --reporter xunit --reporter-option output=nyc/mocha-junit-report.xml",
     "test:realsvc": "lerna run test:realsvc --stream --no-bail --no-sort",
     "test:realsvc:report": "lerna run test:realsvc:report --stream --no-bail --no-sort",
-    "test:report": "npm run test:mocha:report && npm run test:jest:report && echo 'Skipped npm run test:realsvc:report (Issue #6006)'",
+    "test:report": "npm run test:mocha:report && npm run test:jest:report && test:realsvc:report",
```
Comment (anthony-murphy) [inline package.json:65]:
```suggestion
    "test:report": "npm run test:mocha:report && npm run test:jest:report && npm run test:realsvc:report",
```

**Comment by DLehenbauer** [comment]
(Thanks for the assist, @anthony-murphy .  :-)

I think we're going to have some test failures that crept in while the tests were disable.  For example, locally I see tests timing out waiting for the "leader" event.

@MattRakow - I know you were looking at removing the event.  Did that land?

**Comment by anthony-murphy** [comment]
@DLehenbauer are you planning to revert #6062

**Comment by DLehenbauer** [comment]
@anthony-murphy - Essentially.  I thought I might be able to get away with adding a deprecated method that forwards to the new one.

**Comment by anthony-murphy** [comment]
@jatgarg it look like the test `"ReAttach detached container on failed attach"` is failing. probably due to your recent change

**Comment by anthony-murphy** [comment]
ok. i'm running these locally for another change, looks like just the `resubmit` change and the test above are the only current problems

**Comment by jatgarg** [comment]
> @jatgarg it look like the test `"ReAttach detached container on failed attach"` is failing. probably due to your recent change

Let me take a look. But the tests must have passed when I submitted the change.

**Comment by anthony-murphy** [comment]
@jatgarg  the tests were not running due to the  issue here

**Comment by jatgarg** [comment]
I fixed this error: ReAttach detached container on failed attach

**Pull request merged**